content/examples/qubole-cluster.Rmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Using RStudio Server Pro inside of Qubole"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float:
+      collapsed: false
+params:
+  width: 600
+---
+
+## Overview
+
+Qubole users can request [beta access to RStudio Server
+Pro](https://docs.qubole.com/en/latest/user-guide/engines/spark/rstudio_spark.html).
+This allows users to use [`sparklyr`](https://spark.rstudio.com/) to interact
+directly with Spark from within the Qubole cluster.
+
+<img src="/images/deployment/qubole/rstudio-qubole.png" width='800px' align='center'/>
+
+## Advantages and limitations
+
+Advantages:
+
+- Ability for users to connect `sparklyr` directly to Spark within Qubole
+- Provides a high-bandwidth connection between R and the Spark JVM processes
+  because they are running on the same machine
+- Can load data from the cluster directly into an R session since RStudio Server
+  Pro is installed within the Qubole cluster
+- A unique, persistent home directory for each user
+
+Limitations:
+
+- Persistent packages must be managed from the Environments page, not directly
```

Comment:
I had to look at the explanation for this on the other page to understand the context. Maybe change "managed from the Environments page" to "managed using Quoble Environments" to make it more clear?